app/models/site_config.rb

```diff
@@ -62,6 +62,7 @@ class SiteConfig < RailsSettings::Base
   # Onboarding
   field :onboarding_taskcard_image, type: :string, default: "https://practicaldev-herokuapp-com.freetls.fastly.net/assets/staggered-dev.svg"
   field :suggested_tags, type: :array, default: %w[beginners career computerscience javascript security ruby rails swift kotlin]
+  field :suggested_users, type: :array, default: %w[ben jess peter maestromac andy liana]
```

Comment:
I just set this as the default for DEV since that's what we seemed to be falling back to [here](https://github.com/thepracticaldev/dev.to/pull/7637/files#diff-4e05ad0d64e6100656b63ad1e78f32c5L11).

Reply:
I think it's ok to leave this in the PR, but at some point we may to stop adding defaults at all, as they make no sense for other communities. Instead I think we need a sort of onboarding process for community admins that walks them through all of these settings. I'll open an issue for this.

Reply:
@citizen428 there is a card that outlines this at a high level, but we haven't fleshed it out as yet, maybe you could add some detail to it? https://github.com/thepracticaldev/dev.to/projects/15#card-37093809